hawkular-inventory-itest/src/test/groovy/org/hawkular/inventory/rest/test/InventoryITest.groovy

```diff
@@ -1291,6 +1292,90 @@ class InventoryITest extends AbstractTestBase {
         }
     }
 
+    @Test
+    void testSync() {
+        def structure = [
+                type    : "feed",
+                data    : [
+                        id: "sync-feed"
+                ],
+                children: [
+                        resource    : [
+                                [
+                                        data    : [
+                                                id              : "resource",
+                                                resourceTypePath: "resourceType"
+                                        ],
+                                        children: [
+                                                resource: [
+                                                        [
+                                                                data: [
+                                                                        id              : "childResource",
+                                                                        resourceTypePath: "../resourceType"
+                                                                ]
+                                                        ]
+                                                ]
+                                        ]
+                                ]
+                        ],
+                        resourceType: [
+                                [
+                                        data: [
+                                                id  : "resourceType",
+                                                name: "My Resource Type With A Friendly Name"
+                                        ]
+                                ]
+                        ],
+                        metric      : [
+                                [
+                                        data: [
+                                                id                : "metric",
+                                                metricTypePath    : "metricType",
+                                                collectionInterval: 0
+                                        ]
+                                ]
+                        ],
+                        metricType  : [
+                                [
+                                        data: [
+                                                id                : "metricType",
+                                                type              : "GAUGE",
+                                                unit              : "NONE",
+                                                collectionInterval: 0,
```

Comment:
offtopic: perhaps we should rename the `collectionInterval` on metricType entity to `defaultCollectionInterval`

@pavolloffay what do you think? Does it make sense?